zerver/tests/test_decorators.py

```diff
@@ -647,64 +647,43 @@ class TestLoggingErrorException(Exception):
 
         return mock.patch("logging.error", side_effect=TestLoggingErrorException)
 
-    def test_internal_local_clients_skip_rate_limiting(self) -> None:
-        META = {"REMOTE_ADDR": "127.0.0.1"}
+    def check_rate_limit_public_or_user_views(self, remote_addr: str, client_name: str) -> None:
```

Comment:
So this function actually checks that the rate limiting functions **don't** get called, right? It'd be good to have that reflected in the name I think

Reply:
Ohh, this method acquires the functionality of checking either that rate limiting happened or didn't happen in the next commit, so that makes sense then with the name you chose.

Reply:
Maybe just add a sentence in the commit message of `test_decorators: Extract check_rate_limit_public_or_user_views.` that this is about to happen and justifies the general name?